submissions/Roophee/html-movie-seat-booking/style.scss

```diff
@@ -0,0 +1,670 @@
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    }
+
+    &__nav {
+        display: flex;
+        justify-content: center;
+        min-width: 450px;
+        font-size: 1.25rem;
+        order:1;
+
+        &-list {
+            display: flex;
+            justify-content: space-between;
+            list-style: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        &-item {
+            padding: 3px;
+            margin: 5px;
+            border-radius: 6px;
+            color: $font-color;
+            text-align: center;
+
+            & a {
+                text-decoration: none;
+                color: $font-color;
+                text-align: center;
+
+                &:hover {
+                    color: $green;
+                }
+
+                &:active {
+                    color: $darkGreen;
+                }
+            }
+        }
+    }
+
+    &__search {
+        order: 2;
+
+        &-form-input {
+            width: 200px;
+            font-size: 1rem;
+            font-family: inherit;
+            letter-spacing: 2px;
+            font-weight: 300;
+            margin: 0 5px;
+        }
+    }
+
+    &__account {
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+        order: 3;
+
+        &-avatar {
+            margin: 0 10px;
+            display: flex;
+            align-items: center;
+
+            &-img {
+                width: 45px;
+                height: 45px;
+                border-radius: 50%;
+                border: 2px solid $font-color;
+
+                &:hover {
+                    border-color: $green;
+                }
+            }
+        }
+        &-login-btn {
+            width: 70px;
+            height: 30px;
+            font-size: 1.25rem;
+            text-align: center;
+            background-color: $green;
+            border-radius: 10px;
+            color: $font-color;
+            padding:  7px 10px;
+            margin-left: 10px;
+            cursor: pointer;
+
+            &:hover {
+                background-color: #388E3C;
+            }
+
+            &:active {
+                background-color: $darkGreen;
+            }
+        }
+    }
+}
+
+.container__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    align-items: center;
+}
+
+.content__info {
+    z-index: 2;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    padding: 15px 30px;
+    background-color: $black;
+
+    &-header {
+        width: 100%;
+        color: $font-color;
+        margin: 20px;
+        text-align: center;
+    }
+
+    &-poster {
+        margin: 0 30px 0 0 ;
+
+        &-img {
+            width: 320px;
+            height: 450px;
+        }
+    }
+}
+
+.table {
+
+    &-border{
+        color: $font-color;
+        border-bottom: 1px solid $lineColor;
+        margin: 0;
+    }
```

Comment:
Apply these styles to the `tr` once, reduce using similar classes.

Reply:
I desided apply this styles only once on <table> element and used in this appoach selector for all childs whis tags <th> and <td>.
Thanks.